src/megatron/bridge/training/initialize.py

```
@@ -566,6 +566,37 @@ def _initialize_distributed(
+            _fr_env_defaults = {
+                "TORCH_FR_DUMP_TEMP_FILE": _fr_path,
+                "TORCH_NCCL_DEBUG_INFO_TEMP_FILE": _fr_path,
+                "TORCH_NCCL_TRACE_BUFFER_SIZE": str(dist_config.flight_recorder_trace_buffer_size),
+                "TORCH_NCCL_DUMP_ON_TIMEOUT": str(int(dist_config.flight_recorder_dump_on_timeout)),
+                "TORCH_INCLUDE_STACK_TRACE": str(int(dist_config.flight_recorder_include_stack_trace)),
+                "TORCH_INCLUDE_ONLY_ACTIVE": str(int(dist_config.flight_recorder_include_only_active)),
+                "TORCH_NCCL_EXTRA_DUMP_ON_EXEC": str(int(dist_config.flight_recorder_extra_dump_on_exec)),
+            }
+            for _var, _default in _fr_env_defaults.items():
+                if _var in os.environ:
+                    warnings.warn(
```

Ruff (0.15.6)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)